src/controllers/order.controller.js

```
@@ -384,3 +450,64 @@ export const updateOrderItemsController = async (req, res) => {
+
+      case 'extra':
+        // el precio debe venir del ID o hardcodearlo (opcional)
+        const precioExtra = {
+          1: precios.postre,
+          2: precios.ensalada,
+          3: precios.proteina
+        }[item.item_id];
+
+        if (precioExtra) {
+          total += cantidad * precioExtra;
+        }
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)